src/ui/FacetSlider/FacetSlider.ts

```diff
@@ -203,10 +254,19 @@ export class FacetSlider extends Component {
         })
       }
     }),
+
     /**
-     * Specifies a function that will generate the steps for the slider. The function receives the slider boundaries and must return an array of number (the steps).
+     * Specifies a function to generate the steps for the FacetSlider (see {@link FacetSlider.options.steps}. This
+     * function receives the FacetSlider boundaries (see {@link FacetSlider.options.start} and
+     * {@link FacetSlider.options.end}) and must return an array of number (the steps).
```

Comment:
Since "number" is the type, I hesitate to use the plural form. Perhaps I could instead emphasize it, like "array of `number`". What do you think?